### ISSUE TITLE
Could not install when using Ruby 2.5 (x64-mingw32)

### ISSUE THREAD

**Issue by cosmo0920** [tarcieri/cool.io#64]
I tried to install cool.io on Windows with Ruby 2.5.
But I cannot do it.

## Environment

* WIndows 10 Home 1803 64 Bit
* Ruby (from RubyInstaller2) ruby 2.5.0p0 (2017-12-25 revision 61468) [x64-mingw32]

## Log

I tried to bundle install on fluent-plugin-mongo local repository.

```log
PS> ridk enable
PS> bundle install --path vendor/bundle
The dependency cool.io (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for x64-mingw32 but the dependency is only for ruby. To add those platforms to the bundle, run `bundle lock --add-platform ruby`.
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Using rake 12.3.1
Using bson 4.3.0
Using bundler 1.16.2
Fetching cool.io 1.5.3 (x64-mingw32)
Installing cool.io 1.5.3 (x64-mingw32)
Gem::RuntimeRequirementNotMetError: cool.io requires Ruby version < 2.5, >= 2.0.
The current ruby version is 2.5.0.
An error occurred while installing cool.io (1.5.3), and Bundler cannot continue.
Make sure that `gem install cool.io -v '1.5.3' --source 'https://rubygems.org/'`
succeeds before bundling.

In Gemfile:
  fluent-plugin-mongo was resolved to 1.1.2, which depends on
    fluentd was resolved to 1.2.5, which depends on
      cool.io
```

I extracted cool.io-1.5.3-x64-mingw32 and inspected:

```
PS> ls data/lib
PS> 2.0  2.1  2.2  2.3  2.4  cool.io  cool.io.rb  coolio.rb
```

Shared object for Ruby 2.5 is not included in released cool.io gem.

**Comment by surajpasuparthy** [comment]
I have the same problem !. any luck ?

**Comment by repeatedly** [comment]
I forgot to push new gem for it.
Just released v1.5.4 with ruby 2.5 binary.
